Packages/Style/Sources/Button/StateButtonStyle.swift

```diff
@@ -0,0 +1,87 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import SwiftUI
+
+public struct StateButtonStyle: ButtonStyle {
+    public static let maxHeight: CGFloat = 50
+    private let variant: ButtonVariant
+    private let palette: ButtonStylePalette
+
+    public init(_ variant: ButtonVariant, palettee: ButtonStylePalette) {
+        self.variant = variant
+        self.palette = palettee
+    }
+
+    public func makeBody(configuration: Configuration) -> some View {
+        ZStack {
+            RoundedRectangle(cornerRadius: 12)
```

Comment:
```suggestion
            RoundedRectangle(cornerRadius: 12)
```

Use Sizes to avoid hardcode values